doc/_ext/ceph_commands.py

```diff
@@ -108,6 +109,8 @@ def help(self):
             advanced.append('goodchars= ``{}`` '.format(self.goodchars))
         if self.n:
             advanced.append('(can be repeated)')
+        if self.allowempty:
+            advanced.append('(can be empty string)')
```

Comment:
@mchangir since we have a new attribute, does it require changes to docs? or add it to pendingreleasenotes?